interpreter/llvm/src/tools/clang/lib/Serialization/ASTReader.cpp

```diff
@@ -1244,37 +1244,8 @@ bool ASTReader::ReadSourceManagerBlock(ModuleFile &F) {
 /// \brief Attempt to resolve the location based on PP header search.
 /// Find the first match with the longest trailing part.
 static StringRef
-resolveFileThroughHeaderSearch(Preprocessor& PP, StringRef Filename,
-                               llvm::StringMap<std::string>& HSStemMap) {
-  // Locate /a/b/c/d.h given the -I paths, by removing leading subdirectories
-  // until a file is found.
+resolveFileThroughHeaderSearch(Preprocessor& PP, StringRef Filename) {
```

Comment:
I bet also 6545d8c211 e893dfcd36 7e31f48303 8240dcfe4b (hashes from root.git) can be reverted. I'm not sure whether reverting them or not reverting them will result in a more broken state, once you move all zipped headers into the module...

Reply:
That's work in progress, I am intending to do that and then we will see.

Reply:
Btw, it looks like 6545d8c and e893dfc are not active anymore.